lib/v2/screens/sign_up/claim_invite/claim_invite_screen.dart

```diff
@@ -46,7 +39,6 @@ class _ClaimInviteScreenState extends State<ClaimInviteScreen> {
           }
         },
         child: BlocBuilder<SignupBloc, SignupState>(
-          buildWhen: (previousState, currentState) => previousState != currentState,
```

Comment:
@gguij004 Could you confirm me if all these widgets are necessary? for some height fixed or or something similar.

```
            return LayoutBuilder(
              builder: (context, constraint) => SingleChildScrollView(
                child: ConstrainedBox(
                  constraints: BoxConstraints(minHeight: constraint.maxHeight),
                  child: IntrinsicHeight(
                    child: Column(
                      crossAxisAlignment: CrossAxisAlignment.center,
                      mainAxisSize: MainAxisSize.max,
                      children: [
                        Expanded(child: _scannerWidget),
                        const SizedBox(height: 30),
                      ],
                    ),
                  ),
                ),
              ),
            );
```
or is it ok if I just return the QRscanner?
`return _scannerWidget;`

I don't see more difference when I change it than a little height.

Reply:
maybe before was needed cause the screen was more complex, but now thats only the title and the Qr Scan most likely none of that is needed.

Reply:
not needed